api/src/main/java/org/openmrs/api/impl/OrderServiceImpl.java

```diff
@@ -365,7 +365,7 @@ private void discontinueExistingOrdersIfNecessary(Order order) {
 	}
 	
 	private DrugOrder checkDrugOrdersForDiscontinuing(DrugOrder drugOrder1, DrugOrder drugOrder2) {
```

Comment:
Get rid of this method instead and where it is being used change the code to call hasSameOrderableAs()